src/controllers/__tests__/item.test.js

```diff
@@ -180,6 +172,22 @@ describe('Items Endpoints', () => {
 
     test('Should response with an error message when there are no items', async () => {
       await clearDatabase();
+      const signUpResponse = await request(server)
+        .post('/api/auth/signup')
+        .set('Content-Type', 'application/json')
+        .send(mockUser);
+      owner = signUpResponse.body;
+      trueItem.owner = owner;
+      trueItem2.owner = owner;
+      trueItem3.owner = owner;
+      trueItem4.owner = owner;
+      const signInResponse = await request(server)
+        .post(`/api/auth/signin`)
+        .set('Content-Type', 'application/json')
+        .send({ username: mockUser.username, password: mockUser.password });
+
+      [authCookie] = signInResponse.headers['set-cookie'];
```

Comment:
Isn't this the same as what was done in the beforeAll block?

Reply:
because this test needs to use `await clearDatabse()` the owner is being removed  from the database and the tests are not passing

Reply:
because this test needs to use await clearDatabse() the owner is being removed from the database and the tests are not passing

Reply:
Got it. Thanks for clarifying.

Reply:
And I forgot to mention that the reassignment of the `trueItem.owner` is because that the owner id is changing so I had to reassign them all

Reply:
Okay maybe it's worth adding this as an explainer comment in the code.

Reply:
Ok I will do that 
There is another problem popped up and I am trying to solve it now 😅 
I will inform you once I finish